accounts/jobs/daily/collect_payouts.py

```diff
@@ -4,20 +4,25 @@
 
 from django_extensions.management.jobs import DailyJob
 
-from accounts.models import Payout
+from django.db.models.query_utils import Q
+
+from accounts.models import Payout, UserProfileReferralHit
 from accounts.utils import calculate_amount, get_all_active_users
 
 LOGGER = logging.getLogger(__name__)
 
 
 class Job(DailyJob):
+    """
+    Daily job
+    """
+
```

Comment:
@autn usually what I like to do with jobs is to call a utility function in the `execute` method - this makes it a little easier to test. Do we currently have a test covering this logic?

Reply:
@YPCrumble we have a test for this, you can see `accounts.tests.PayoutAmountTest`

Reply:
👍